app/controllers/courses/age_range_controller.rb

```diff
@@ -1,17 +1,11 @@
 module Courses
   class AgeRangeController < ApplicationController
+    include CourseBasicDetailConcern
     decorates_assigned :course
     before_action :build_course, only: %i[edit update]
-    before_action :build_provider, :build_new_course, only: %i[new continue]
-
-    def new; end
 
     def continue
-      redirect_to confirmation_provider_recruitment_cycle_courses_path(
-        params[:provider_code],
-        params[:recruitment_cycle_year],
-        course_params,
-      )
+      redirect_to next_step(current_step: :age_range)
```

Comment:
The other `continue` actions render out errors. Should this one too?

Reply:
The error handling across all the continues need a bit of work, I can make this look the same for now but none of them really connect to anything yet (didn't want to make this PR bigger)